internal/promotion/runner/builtin/schemas/git-open-pr-config.json

```
@@ -15,8 +15,8 @@
     },
     "provider": {
       "type": "string",
-      "description": "The name of the Git provider to use. Currently only 'github', 'gitlab', 'gitea' and 'azure' are supported. Kargo will try to infer the provider if it is not explicitly specified.",
-      "enum": ["github", "gitlab", "azure", "gitea"]
+      "description": "The name of the Git provider to use. Currently 'github', 'gitlab', 'bitbucket', 'gitea' and 'azure' are supported. Kargo will try to infer the provider if it is not explicitly specified.",
+      "enum": ["github", "gitlab", "azure", "gitea", "bitbucket"]
```

Same as [here](https://github.com/akuity/kargo/pull/3209/files#r2035953690)

```suggestion
      "description": "The name of the Git provider to use. Currently 'azure', 'bitbucket', 'gitea', 'github', and 'gitlab' are supported. Kargo will try to infer the provider if it is not explicitly specified.",
      "enum": ["azure", "bitbucket", "gitea", "github", "gitlab"]
```